hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/HoodieSparkCopyOnWriteTable.java

```diff
@@ -112,12 +112,12 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> upsert(HoodieEngineContext c
 
   @Override
   public HoodieWriteMetadata<HoodieData<WriteStatus>> insert(HoodieEngineContext context, String instantTime, HoodieData<HoodieRecord<T>> records) {
-    return new SparkInsertCommitActionExecutor<>((HoodieSparkEngineContext)context, config, this, instantTime, records).execute();
+    return new SparkInsertCommitActionExecutor<>((HoodieSparkEngineContext) context, config, this, instantTime, records).execute();
   }
 
   @Override
   public HoodieWriteMetadata<HoodieData<WriteStatus>> bulkInsert(HoodieEngineContext context, String instantTime, HoodieData<HoodieRecord<T>> records,
-      Option<BulkInsertPartitioner> userDefinedBulkInsertPartitioner) {
+                                                                 Option<BulkInsertPartitioner> userDefinedBulkInsertPartitioner) {
```

Comment:
can you revert changes in these files if these are just unintentional formatting changes.

Reply:
yes, I thought they should be removed.